packages/flutter_tools/test/web.shard/expression_evaluation_web_test.dart

```diff
@@ -53,6 +53,19 @@ void main() {
       );
     }
 
+    testWithoutContext('flutter run expression evaluation - can evaluate method from dependent library after hot reload', () async {
```

Comment:
How much duplicated code is there under `web.shard`?

Reply:
Not as much as you'd think, since web expression evaluation still works a little differently (ints are doubles, unused depedencies aren't loaded). I don't love the current set up